bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -1328,18 +1326,8 @@ void checkAllLedgers() throws BKException, IOException, InterruptedException {
             final CompletableFuture<Void> processFuture = new CompletableFuture<>();
 
             Processor<Long> checkLedgersProcessor = (ledgerId, callback) -> {
-                try {
-                    if (!ledgerUnderreplicationManager.isLedgerReplicationEnabled()) {
-                        LOG.info("Ledger rereplication has been disabled, aborting periodic check");
-                        FutureUtils.complete(processFuture, null);
-                        return;
-                    }
-                } catch (ReplicationException.NonRecoverableReplicationException nre) {
-                    LOG.error("Non Recoverable Exception while reading from ZK", nre);
-                    submitShutdownTask();
-                    return;
```

Comment:
why is this removed / where NonRecoverableReplicationException is handled now?

Reply:
it's handled where `notifyLedgerReplicationStatusChanged` is called